src/library/scala/math/Numeric.scala

```diff
@@ -168,10 +162,6 @@ object Numeric {
   trait DoubleIsFractional extends DoubleIsConflicted with Fractional[Double] {
```

Comment:
If `DoubleAsIfIntegral` is being removed, `DoubleIsConflicted` can probably be merged into `DoubleIsFractional`, as it's no longer conflicted (same for `FloatIsFractional`/`FloatIsConflicted`)

Reply:
Sounds good 👍